app-media-image-capture.html

```diff
@@ -191,118 +202,144 @@
 
-          /** @see https://www.w3.org/TR/image-capture/#PhotoSettings */
+          /** @see https://w3c.github.io/mediacapture-image/#mediatracksettings-members */
           iso: {
             type: Number,
             value: null
           },
 
-          /** @see https://www.w3.org/TR/image-capture/#PhotoSettings */
-          redEyeReduction: {
-            type: Boolean,
-            value: null
-          },
-
           /**
            * @type {Polymer.AppMedia.MeteringMode}
-           * @see https://www.w3.org/TR/image-capture/#PhotoSettings
+           * @see https://w3c.github.io/mediacapture-image/#mediatracksettings-members
            */
           focusMode: {
             type: String,
             value: null
           },
 
-          /** @see https://www.w3.org/TR/image-capture/#PhotoSettings */
+          /** @see https://w3c.github.io/mediacapture-image/#mediatracksettings-members */
           pointsOfInterest: {
             type: Array,
             value: null
           },
 
-          /** @see https://www.w3.org/TR/image-capture/#PhotoSettings */
+          /** @see https://w3c.github.io/mediacapture-image/#mediatracksettings-members */
           brightness: {
             type: Number,
             value: null
           },
 
-          /** @see https://www.w3.org/TR/image-capture/#PhotoSettings */
+          /** @see https://w3c.github.io/mediacapture-image/#mediatracksettings-members */
           contrast: {
             type: Number,
             value: null
           },
 
-          /** @see https://www.w3.org/TR/image-capture/#PhotoSettings */
+          /** @see https://w3c.github.io/mediacapture-image/#mediatracksettings-members */
           saturation: {
             type: Number,
             value: null
           },
 
-          /** @see https://www.w3.org/TR/image-capture/#PhotoSettings */
+          /** @see https://w3c.github.io/mediacapture-image/#mediatracksettings-members */
           sharpness: {
             type: Number,
             value: null
           },
 
-          /** @see https://www.w3.org/TR/image-capture/#PhotoSettings */
+          /** @see https://w3c.github.io/mediacapture-image/#mediatracksettings-members */
           zoom: {
             type: Number,
             value: null
           },
 
-          /** @see https://www.w3.org/TR/image-capture/#PhotoSettings */
-          imageHeight: {
-            type: Number,
-            value: null
-          },
-
-          /** @see https://www.w3.org/TR/image-capture/#PhotoSettings */
-          imageWidth: {
-            type: Number,
-            value: null
-          },
-
-          /**
-           * @type {Polymer.AppMedia.FillLightMode}
-           * @see https://www.w3.org/TR/image-capture/#PhotoSettings
-           */
-          fillLightMode: {
-            type: String,
+          /** @see https://w3c.github.io/mediacapture-image/#mediatracksettings-members */
+          torch: {
+            type: Boolean,
```

Comment:
After further investigation, I believe this to be the best way forward for normalized behavior between 1.x and 2.x.